cav_msgs/msg/TrajectoryPlanPoint.msg

```diff
@@ -0,0 +1,22 @@
+# TrajectoryPlanPoint.msg
+
+# This message describes a 2D point on plugin's trajectory plan
+# It is assmued that this point is in the MAP frame
+
+# X coordinate of the trajectory plan point
+float64 x
+
+# Y coordinate of the trajectory plan point
+float64 y
+
+# Desired time when reaching the trajectory plan point
+uint64 timestamp
```

Comment:
I think having the term target_time here would be clearer. This could be confused with current time.